commands/playground/youtubeQueue.js

```diff
@@ -0,0 +1,30 @@
+const ytdl = require("ytdl-core");
+const {youtubeSearch}  = require("./../music/youtubeSearch");
+
+const youtubeQueue = (queue, connection, voiceChannel) => {
```

Comment:
not sure what types these params are.  consider Typescript

Reply:
Ill consider it for next project